### PR TITLE
Add pydocstyle linter

### PR DESCRIPTION
**Pull request by quitterie-lcs**
## Purpose

`pydocstyle` is a docstring linter that checks compliance for docstrings with Python conventions.

## Proposal

- Configure pydocstyle 
- Fix errors

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python # noqa: D100
```
Comment (jmaupetit) [inline setup.py:1]: Who's complaining here? About What?

File: setup.cfg [hunk 2]
```diff
@@ -111,6 +112,11 @@ exclude =
     node_modules,
     */migrations/*
 
+[pydocstyle]
+convention = google
+match = .*\.py
+match_dir = ^(?!(tests)).*
```
Comment (jmaupetit) [inline setup.cfg:118]: Can you instead restrict the analysis to `src/ralph`?

File: setup.py (hunk 1 above)
Comment (quitterie-lcs) [reply setup.py:1]: Not necessary anymore

**Review by jmaupetit — APPROVED** [review]

**Pull request merged**